modules/govuk_jenkins/templates/jobs/ask_export.yaml.erb

```
@@ -0,0 +1,81 @@
+---
+- scm:
+    name: govuk-ask-export
+    scm:
+      - git:
+          url: git@github.com:alphagov/govuk-ask-export.git
+          branches:
+            - master
+
+- job:
+    name: govuk-ask-export
+    display-name: GOV.UK Ask Export
+    project-type: freestyle
+    <% prod_description = "This job exports responses from the Ask service so that they can be used by the relevant departments" %>
+    <% integration_description = "This is for testing so doesn't run unless you tell it to" %>
+    description: <%= @run_daily ? prod_description : integration_description %>
```

```suggestion
    <% daily_description = "This job performs a daily export of responses from the Ask service using the $smart_survey_config Smart Survey Configuration" %>
    <% on_demand_description = "This job performs on demand exports of responses from the Ask service using the $smart_survey_config Smart Survey Configuration" %>
    description: <%= @run_daily ? daily_description : on_demand_description %>
```

Would suggest we don't couple this too much to integration / prod since it's done off a variable that isn't based on that.

---

The only thing we changed here was the string interpolation because it's an erb file we used the `#{blah blah}` syntax rather than `$` please let us know if this is wrong :)